exercises/concept/boutique-inventory/.meta/exemplar.rb

```diff
@@ -2,18 +2,18 @@ class BoutiqueInventory
   def initialize(items)
     @items = items
   end
-  
+
   def item_names
     items.map { |item| item[:name] }.sort
   end
- 
+
   def cheap
     items.select { |item| item[:price] < 30 }
   end
- 
+
   def out_of_stock
-    items.select do |item| 
-      item[:quantity_by_size].none {|size, quantity| quanity > 0 }
+    items.select do |item|
+      item[:quantity_by_size].none {|size, quantity| quantity > 0 }
```

Comment:
This intrigues me, shouldn't this have been caught by a CI check that runs tests on all exemplar solutions?

Reply:
There's even more mistakes. The function is called `none?`, not `none`. That means that `out_of_stock` is likely never hit.

Reply:
```suggestion
      item[:quantity_by_size].none? {|size, quantity| quantity > 0 }
```

Reply:
Got it. After that change, the tests pass, I checked manually. But it looks like the CI check is for practice exercises only.